front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -313,6 +339,56 @@ export function AgentMessage({
   }
 }
 
+function Citations({
+  activeReferences,
+}: {
+  activeReferences: { index: number; document: RetrievalDocumentType }[];
+}) {
+  activeReferences.sort((a, b) => a.index - b.index);
+  return (
+    <div className="w-[110%] overflow-x-auto scrollbar-hide sm:w-[120%]">
```

Comment:
Why 110% ?

Reply:
If your screen is very large them it'll be cut at 110% content size right?

I think it should overflow all the way to the end of the screen no? Maybe not... a question for design I presume.

But the 110% feels a bit weird?

Reply:
I agree, this was a compromise solution after spending ~10mn trying to cleanly make it go to the end 
I'll spend a bit more time looking into it